### PR TITLE
fix config by adding in CLOUD_PROVIDER

### PR DESCRIPTION
**Pull request by wajihyassine**

### CODE REVIEW

File: modules/turbinia/templates/turbinia.conf.tpl [hunk 1]
```diff
@@ -2,6 +2,7 @@
 INSTANCE_ID         = 'turbinia-${turbinia_id}'
 STATE_MANAGER       = 'Datastore'
 TASK_MANAGER        = 'PSQ'
+CLOUD_PROVIDER      = 'GCP'
```
Comment (aarontp) [inline modules/turbinia/templates/turbinia.conf.tpl:5]: I think some other config variables have been added as well (e.g. for API server).  Can you double check if there are other ones that we need to add?

Comment (wajihyassine) [reply modules/turbinia/templates/turbinia.conf.tpl:5]: Yes np, done

Comment (aarontp) [reply modules/turbinia/templates/turbinia.conf.tpl:5]: Cool, thanks!

**Review by aarontp — APPROVED** [review]
LGTM

**Pull request merged**